extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -325,6 +325,7 @@ export abstract class AzureAuth implements vscode.Disposable {
 			return tenants;
 		} catch (ex) {
 			console.log(ex);
+			console.log(JSON.stringify(ex?.response?.data, undefined, 2));
```

Comment:
Are we confident this response data won't contain sensitive info that we don't want in the logs?  Perhaps like an auth token or something like that?

Reply:
Yeah this shouldn't log anything sensitive - and worst case if something on the AAD side messes up, the token will only have a validity of 15 minutes to an hour.